run_scripts/4_coda_add.sh

```diff
@@ -27,6 +27,7 @@ DATASETS=(
     "COVID19_KE_Urban_s01e04"
     "COVID19_KE_Urban_s01e05"
     "COVID19_KE_Urban_s01e06"
+    "COVID19_KE_Urban_s01e07"
```

Comment:
we should probably comment this so that we don't create the doc in firestore until we are sure we need it?